mungegithub/mungers/block_paths.go

```diff
@@ -66,6 +67,10 @@ func (b *BlockPath) RequiredFeatures() []string { return []string{} }
 
 // Initialize will initialize the munger
 func (b *BlockPath) Initialize(config *github.Config, features *features.Features) error {
+	return b.loadConfig()
+}
+
+func (b *BlockPath) loadConfig() error {
```

Comment:
If an update callback returns an error mungegithub exits anyways so the fatal calls are the same as returning an error. I'll switch them to returning an error though for consistency. If someone pushed a config that updates the `block-path-config` option and the option points to an invalid file then mungegithub would start crashlooping. I updated the munger to log an error instead of crashing if loadConfig fails on a reload.